includes/Modules/Analytics.php

```diff
@@ -1325,29 +1331,10 @@ private function get_analyticsreporting_service() {
 	 */
 	protected function setup_services( Google_Site_Kit_Client $client ) {
 		$google_proxy = new Google_Proxy( $this->context );
-
-		// Create an analytics provisioning service that makes requests to the proxy.
-		$analytics_provisioning_service = new Google_Service_Analytics( $client, $google_proxy->url() );
-
-		// Use a custom provisioning method that accepts site id and secret.
-		$analytics_provisioning_service->provisioning = new Proxy_Provisioning(
-			$analytics_provisioning_service,
-			$analytics_provisioning_service->serviceName, // phpcs:ignore WordPress.NamingConventions.ValidVariableName
-			'provisioning',
-			array(
-				'methods' => array(
-					'createAccountTicket' => array(
-						'path'       => 'provisioning/createAccountTicket',
-						'httpMethod' => 'POST',
-						'parameters' => array(),
-					),
-				),
-			)
-		);
 		return array(
 			'analytics'             => new Google_Service_Analytics( $client ),
 			'analyticsreporting'    => new Google_Service_AnalyticsReporting( $client ),
-			'analyticsprovisioning' => $analytics_provisioning_service,
+			'analyticsprovisioning' => new Google_Service_AnalyticsProvisioning( $client, $google_proxy->url() ),
```

Comment:
+1 for using this name. Original proposal was `Google_Proxy_Service_AnalyticsProvisioning`, but I think this one works well. Without the proxy, this wouldn't be a separate class anyway and we'd use `Google_Service_Analytics`.

Reply:
👍